terragrunt/aws/ecr/iam_oidc.tf

```
@@ -98,4 +98,25 @@ data "aws_iam_policy_document" "ecr_image_manage" {
     ]
     resources = ["*"]
   }
+}
+
+data "aws_iam_policy_document" "pr_review_env_policy_document" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
+    ]
+
+    resources = [
+      "arn:aws:lambda:${var.region}:${var.account_id}:function:pr-review-env-*"
```

This needs to be a CloudWatch ARN pattern like this:
```suggestion
      "arn:aws:logs:${var.region}:${var.account_id}:*"
```